src/numeric/decision_rules.jl

```diff
@@ -1,9 +1,43 @@
-import splines
+abstract AbstractDecisionRule{S,T}
 
-abstract AbstractDecisionRule
-abstract AbtsractCubicSplineDR <: AbstractDecisionRule
+# abstract AbstractCachedDecisionRule{S,T} <: AbstractDecisionRule{S,T}
+# we don't implement that using subtypes anymore
+
+type DecisionRule{S,T} <: AbstractDecisionRule{S,T}
+    grid_exo::S
+    grid_endo::T
+    n_x::Int # number of values (could be a method if there was a nice method name)
+    coefficients::Array{Array{Float64}}
```

Comment:
Do we know the dimensionality of the inner Arrays here? If so we should add that info so the compiler knows what's going on